profiler/test/Datadog.Profiler.Native.Tests/ExporterTest.cpp

```diff
@@ -0,0 +1,111 @@
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2022 Datadog, Inc.
+
+#include "gtest/gtest.h"
+
+#include "Exporter.h"
+#include "Profile.h"
+#include "Tags.h"
+
+#include "shared/src/native-src/dd_filesystem.hpp"
+
+namespace libdatadog {
+
+Profile CreateEmptyProfile()
+{
+    return Profile({{"cpu", "nanosecond"}}, "RealTime", "Nanoseconds", "my app");
+}
+
+TEST(ExporterTest, EnsureCrashOnDebug)
+{
+    auto builder = Exporter::ExporterBuilder();
+    std::unique_ptr<Exporter> exporter;
+    EXPECT_DEBUG_DEATH({
+        exporter = builder.Build();
+    },
+                       "")
```

Comment:
maybe the EXPECT_DEBUG_DEATH could be written on 1 line

Reply:
✔️